### DISCUSSION TITLE
Adding custom chrome buttons

### DISCUSSION THREAD

**Discussion by GeertvanHorrik** [fluentribbon/Fluent.Ribbon#890] [Q&A] [answered]
I just noticed this theme switcher on the visual studio installer:

![image](https://user-images.githubusercontent.com/1246444/104091190-24b20180-527c-11eb-8f7e-d7a7025bf973.png)

I'd like to add something similar to our apps based on Fluent.Ribbon and was wondering whether this is supported by either Fluent.Ribbon or Window chrome in ControlzEx.

On some apps, we are using the Fluent window without a ribbon (we don't really need a window but like the whole look and feel and integration so still use the ribbon window). So we need actual window chrome buttons, not just ribbon buttons.

**Comment by batzen** [comment, marked as answer]
Something like? 😉
```
    <Fluent:RibbonWindow.WindowCommands>
        <Fluent:WindowCommands>
            <Button Content="Test" Style="{DynamicResource Fluent.Ribbon.Styles.WindowCommands.Button}" />
        </Fluent:WindowCommands>
    </Fluent:RibbonWindow.WindowCommands>
```

![image](https://user-images.githubusercontent.com/1971030/104091356-30ea8e80-527d-11eb-8630-cfce4939176a.png)

**Reply by GeertvanHorrik** [reply]
Fantastic, thank you very much. Sorry for my late reply, I didn't get any notifications via octobox.
